### PR TITLE
[12.0][MIG] account_invoice_tax_note

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@acsonefho @sbidoul

**Review by acsonefho — APPROVED** [review]
LGTM (code only)

File: account_invoice_tax_note/models/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    @api.multi
+    def _get_account_tax_groups_with_notes(self):
+        self.ensure_one()
+        tax_groups = self.mapped("tax_line_ids.tax_id.tax_group_id")
+        tax_groups = tax_groups.filtered(lambda g: g.report_note)
+        return tax_groups
```
Comment (tbaden) [inline account_invoice_tax_note/models/account_invoice.py:14]: you could shorten it by one line if you use:
```diff
-        tax_groups = tax_groups.filtered(lambda g: g.report_note)
-        return tax_groups
+        return tax_groups.filtered(lambda g: g.report_note)
```

**Review by tbaden — APPROVED** [review]
thanks for the work 👍 
I did a code review: LGTM

**Review by tafaRU — APPROVED** [review]
Only techincal review.

**Pull request merged**